src/main/java/org/checkstyle/autofix/recipe/Header.java

```diff
@@ -0,0 +1,140 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// checkstyle-openrewrite-recipes: Automatically fix Checkstyle violations with OpenRewrite.
+// Copyright (C) 2025 The Checkstyle OpenRewrite Recipes Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package org.checkstyle.autofix.recipe;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.checkstyle.autofix.parser.CheckstyleViolation;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.Tree;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaSourceFile;
+import org.openrewrite.java.tree.Space;
+
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+
+public class Header extends Recipe {
+
+    private static final String HEADER_LITERAL = "header";
+
+    private List<CheckstyleViolation> violations;
+    private Configuration headerConfig;
+
+    public Header(List<CheckstyleViolation> violations, Configuration headerConfig) {
+        this.violations = violations;
+        this.headerConfig = headerConfig;
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Header recipe";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Adds headers to Java source files when missing.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        final String licenseHeader = getLicenseHeader(headerConfig);
+        return new HeaderVisitor(violations, licenseHeader);
+    }
+
+    private static String getLicenseHeader(Configuration child) {
+        final String result;
+        final String[] propertyNames = child.getPropertyNames();
+        final boolean hasHeaderProperty = Arrays.asList(propertyNames).contains(HEADER_LITERAL);
+        try {
+            if (hasHeaderProperty) {
+                result = child.getProperty(HEADER_LITERAL);
+            }
+            else {
+                final String headerFile = child.getProperty("headerFile");
```

Comment:
could you move `"headerFile"` to a static constant like you did for the other property? It is to have both properties located in one place and follow similar naming pattern